drape_frontend/gui/skin.cpp

```diff
@@ -90,9 +90,9 @@ class ResolverParser
     else if (attr == "vertical" || attr == "horizontal")
     {
       if (m_element == Element::Anchor)
-        m_resolver.AddAnchor(ParseValueAnchor(value));
+        m_resolver.AddAnchor(ParseValueAnchor(std::string_view{value}));
       else if (m_element == Element::Relative)
-        m_resolver.AddRelative(ParseValueAnchor(value));
+        m_resolver.AddRelative(ParseValueAnchor(std::string_view{value}));
```

Comment:
nit: Take ```std::string_view value``` in Parse definition.

Reply:
No, there are two cases above that will work faster without a temporary string_view. Now it is created only when necessary.

Reply:
Ok, then write simple ParseValueAnchor(value)

Reply:
Done.